_posts/2022-05-26-welcome-to-jekyll.markdown

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
This file isn't needed, eh?

Reply:
Yes! I'll remove it. This file is created whenever someone creates a new Jekyll project.